src/include/common/types/types.h

```diff
@@ -274,6 +277,11 @@ class LogicalType {
     KUZU_API static std::vector<LogicalType> copy(const std::vector<LogicalType*>& types);
 
     static LogicalType ANY() { return LogicalType(LogicalTypeID::ANY); }
+    static LogicalType ANY(PhysicalTypeID physicalType) {
```

Comment:
Comment here say this is a temporary hack and this interface is NOT supposed to be used anywhere else as we should get rid of this later.

Reply:
updated